frontend/.vscode/launch.json

```diff
@@ -26,6 +26,24 @@
         "CI": "true"
       },
       "disableOptimisticBPs": true
+    },
```

Comment:
FYI I'm using jest-runner and Alejandro is using vscode-jest to assist in running tests. If you are interested one of us can give you a demo.

Reply:
that would a cool friday chat!